openmdao/core/group.py

```diff
@@ -53,6 +53,10 @@ class Group(System):
         Dictionary of input_name: (output_name, src_indices) connections.
     _group_inputs : dict
         Mapping of promoted names to certain metadata (src_indices, units).
+    _static_group_inputs : dict
+        Group inputs added outside of setup/configure.
+    _post_setup_group_inputs : dict
```

Comment:
This name confuses me a little. At first, I thought it meant after the user calls prob.setup.  At second, I thought that configure is technically "post setup" too.  Maybe pre_configure is better?

Reply:
fixed